main/data/default_encryption_dialog.ui

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.22.2 -->
+<interface>
+  <requires lib="gtk+" version="3.6"/>
+  <object class="GtkDialog" id="dialog">
+    <property name="can_focus">True</property>
+    <property name="modal">True</property>
+    <property name="default_width">320</property>
+    <property name="default_height">260</property>
+    <child type="titlebar">
+      <placeholder/>
+    </child>
+    <child>
+      <object class="GtkBox">
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="GtkBox">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="orientation">vertical</property>
+            <child>
+              <object class="GtkLabel" id="default_encryption_warning_label">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="label" translatable="yes">No default end-to-end encryption method has been previously selected for this conversation. XMPP defines some XEP for end-to-end encryption that Dino supports.
+
+It is strongly recommended that one of the following end-to-end encryption methods below is selected before sending a message.
+
+TAKE INTO ACCOUNT UNENCRYPTED CONVERSATIONS COULD BE READ BY THIRD PARTIES.
+
+These settings can be later changed from the Settings menu.</property>
+                <property name="wrap">True</property>
+              </object>
+            </child>
+            <child>
+              <object class="GtkCheckButton" id="omemo">
+                <property name="label" translatable="yes">OMEMO (XEP-0384)</property>
```

Comment:
XEP is a technical term and an abbreviation at that. In addition with the number, that can easily confuse users without technical background. If they don't understand it, they can ignore it, choose an encryption method randomly and hope for the best. Or not encrypt and deal with the bad conscience of having chosen the discouraged option. Or search on the internet and read long XEPs that are meant for developers, not end users who just want to chat with their grandchildren.

Dino's simple interface is aimed at such users, so we have to expect a considerable amount of people who don't care much about the technical details and will choose an encryption method if it doesn't get in their way.

Everything those people need to know is that OMEMO will do everything automatically for them while for OpenPGP, they have to set up the keys with an external program.

If someone is really interested in the details, they can just type "OMEMO" in a search engine and will get a lot of information, both the XEP number and simpler descriptions targeted at end users.

```suggestion
                <property name="label" translatable="yes">OMEMO (automatic setup)</property>
```

Reply:
> XEP is a technical term and an abbreviation at that. In addition with the number, that can easily confuse users without technical background.

My intention was to make clear Dino [currently](https://github.com/dino/dino/wiki/Supported-XEPs) only supports [XEP-0027](https://xmpp.org/extensions/xep-0027.html), which is considered obsolete, supposedly in favor of more modern XEPs such as [XEP-0373](https://xmpp.org/extensions/xep-0373.html) or [XEP-0374](https://xmpp.org/extensions/xep-0374.html).

But I agree it sounds _too_ technical for non-tech-savvy users, thus I find your suggestion better. I just hope it does not fail to maintain the neutrality between choosing either OMEMO or OpenPGP, as this has been a reportedly controversial topic within the Dino community.

Reply:
I see. I think that users with sufficient knowledge and interested in the supported XEPs will be able to [look it up on the website](https://dino.im/#features). Therefore, personally, I wouldn't add that information in the UI. But if you would like to do so, here are two ideas:

* Make the option texts links to the related XEPs, though I'm not sure if this is possible with GTK
  * [OMEMO](https://xmpp.org/extensions/xep-0384.html) (automatic setup)
  * [OpenPGP](https://xmpp.org/extensions/xep-0027.html) (external setup required)
*  Add a question mark icon in the corner of the dialogue that provides a short technical explanation when hovered or clicked on

I wrote my suggestions intentionally in a neutral way, i.e. the texts in parentheses describe only facts. Those are chosen in a way to be relevant for less technical users and easy to understand. So I think it's fine regarding neutrality, but of course it's just suggestions from my side and in the end you decide :slightly_smiling_face:

Reply:
>- Make the option texts links to the related XEPs, though I'm not sure if this is possible with GTK 

I am not sure, either. However, I agree tech-savvy users would look up that information somewhere else.

> - Add a question mark icon in the corner of the dialogue that provides a short technical explanation when hovered or clicked on

Sounds like a good suggestion. I will have a look at it ASAP.

> I wrote my suggestions intentionally in a neutral way, i.e. the texts in parentheses describe only facts.

And I also think these decisions are neutral. Considering how many PRs related to this topic have already been either rejected or ignored because of neutrality reasons between OMEMO and OpenPGP, I just wanted to make sure this PR is not affected by this.